common/src/main/scala/sigmastate/VersionContext.scala

```diff
@@ -20,6 +20,10 @@ case class VersionContext(activatedVersion: Byte, ergoTreeVersion: Byte) {
 
   /** @return true, if JIT version of Ergo protocol was activated. */
   def isJitActivated: Boolean = activatedVersion >= JitActivationVersion
+
+  /** @return true, if ErgoTree version >= JitActivationVersion. */
+  def isEvaluateErgoTreeUsingJIT: Boolean =
```

Comment:
Sounds like a tree of a version where isEvaluateErgoTreeUsingJIT is not evaluated used JIT costing, which is not true.

Reply:
"isEvaluateErgoTreeUsingJIT" sounds like whether a tree should be evaluated using JIT or not, but in reality the function checks that tree version is above threshold (and it is not related to JIT evaluation in general)

Reply:
I would like to keep the method and use it where necessary.
So, if mentioning JIT is misleading here I can rename it to `isErgoTreeVersionGreaterV1`. 
@kushti Is it ok?

Reply:
@aslesarenko yes, sounds precise now

Reply:
Done